tests/integration/main.rs

```diff
@@ -119,6 +130,13 @@ fn generate_entity_with_values(prop_string: String, prop_int: i64) -> TestEntity
     }
 }
 
+fn generate_child(parent: &TestEntity) -> TestEntityChild {
+    TestEntityChild {
+        key: IdentifierId::id(parent.key.id, id![None]),
```

Comment:
Here is thing I mentioned over last coffee, would be nice to figure out a better way to do this.
It works now whit only one level of ancestors, but not really convenient, here I would like to be able to just write something like:

`key: parant.key.child_with_id(None)`

Reply:
Maybe something like

* `parent.child_with_id()` trait blanket impl on anything that implements `Key`
and / or
* ` id![parent, None]`

Actually it might be better this way, because now you can only put `None` on the last thing?

Reply:
yes, will start with fixing the macros so that they will take child last, just as everything else. That will come in my next PR